YarnSpinnerConsole/Main.cs

```diff
@@ -420,7 +425,8 @@ static internal Dialogue CreateDialogue(ExecutionOptions options, ConsoleRunnerI
                     {
                         dialogue.LogErrorMessage("ASSERTION FAILED: " + parameters[1].AsString);
                     }
-                    else {
+                    else
+                    {
```

Comment:
I accidentally hit auto-format in this file. (Sorry.)  
Is this alright or should I apply a specific formatting style?